content/en/blog/2023/experimental-dual-stack/index.md

```diff
@@ -0,0 +1,209 @@
+native IP family going forward). To do this, we initially had to duplicate Envoy configuration for listeners, clusters,
+routes and endpoints. Given that many people already experience Envoy memory and CPU consumption issues, early feedback
+wanted us to completely re-evaluate this approach. Many proxies transparently handle outbound dual-stack traffic regardless
+of how the traffic was originated. Much of the earliest feedback was to implement the same behavior in Istio and Envoy.
+
+## Redefining Dual Stack Support
+
+Much of the feedback provided by the community for the original RFC was to update Envoy to better support dual-stack use cases
+internally instead of supporting this within Istio. This has led us to a [new design](https://docs.google.com/document/d/15LP2XHpQ71ODkjCVItGacPgzcn19fsVhyE7ruMGXDyU/edit?usp=sharing)
+where we have taken lessons learned as well as feedback and have applied them to fit a simplified design.
+
+## Support for Dual Stack in Istio 1.17
+
+We have worked with the Envoy community to address numerous concerns which is a reason why dual-stack enablement has
+taken us a while to implement. We have implemented [matched IP Family for outbound listener](https://github.com/envoyproxy/envoy/issues/16804)
+and [supported multiple addresses per listener](https://github.com/envoyproxy/envoy/issues/11184). Alex Xu has also
+been working fervently to get long outstanding issues resolved, with the ability for Envoy to have a
+[smarter way to pick endpoints for dual-stack](https://github.com/envoyproxy/envoy/issues/21640). Some of these improvements
+to Envoy, such as the ability to [enable socket options on multiple addresses](https://github.com/envoyproxy/envoy/pull/23496),
+have landed in the Istio 1.17 release (e.g. [extra source addresses on inbound clusters](https://github.com/istio/istio/pull/41618)).
+
+The Envoy API changes made by the team can be found at their site at [Listener addresses](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/listener/v3/listener.proto.html?highlight=additional_addresses) and [bind config](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/address.proto#config-core-v3-bindconfig). Making sure we can have proper support at both the downstream and upstream connection for Envoy is important for realizing
+dual-stack support.
+
+In total the team has submitted over a dozen PRs to Envoy and are working on at least a half dozen more to make Envoy adoption of
+dual stack easier for Istio.
+
+Meanwhile, on the Istio side you can track the progress in [Issue #40394](https://github.com/istio/istio/issues/40394).
+Progress has slowed down a bit lately as we continue working with Envoy on various issues, however, we are happy to
+announce experimental support for dual stack in Istio 1.17!
+
+## Enabling Dual Stack in Istio 1.17
+
+1. Enable dual stack experimental support on Istio 1.17.0+ with the following:
```

Comment:
1.17 also supports HBONE, how is dual stack working w/ HBONE?

Reply:
yep, givem HBONE is at experimental phase, dual stack work w/HBONE is not covered in this blog.

Reply:
ok, sounds good not  mentioning it.